specs/capella/beacon-chain.md

```diff
@@ -318,6 +320,24 @@ def is_partially_withdrawable_validator(validator: Validator, balance: Gwei) ->
     return has_eth1_withdrawal_credential(validator) and has_max_effective_balance and has_excess_balance
 ```
 
+### Misc
+
+#### Modified `compute_fork_version`
+
+```python
+def compute_fork_version(epoch: Epoch) -> Version:
+    """
+    Return the fork version at the given ``epoch``.
+    """
+    if epoch >= CAPELLA_FORK_EPOCH:
+        return CAPELLA_FORK_VERSION
+    if epoch >= BELLATRIX_FORK_EPOCH:
+        return BELLATRIX_FORK_VERSION
+    if epoch >= ALTAIR_FORK_EPOCH:
+        return ALTAIR_FORK_VERSION
+    return GENESIS_FORK_VERSION
+```
```

Comment:
I'd kinda hesitate to say where we should put this helper on since the core protocol doesn't need it. Other options are:
- `sync-protocol.md`: seems "correct" way, but then we will need to update sync protocol in other directories.
- `fork.md`: might be better than "beacon-chain.md"?

/cc @ralexstokes @djrtwo what do you think?

Reply:
Moved it to `fork.md` for now, it is where the various `_FORK_EPOCH` and `_FORK_VERSION` are introduced, and the helper functions fits well in its theme. Also, the other functions in `fork.md` are not used in `beacon-chain.md` either, so it fits naturally :)